From 45fd3da3c2bd2b22bdf3db1a9a00250d67f250d1 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 20 Sep 2012 17:29:51 +0800
Subject: [PATCH 068/304] ENGR00225217 overlay config: adjust the
 autobrightness array

The following array should have size one greater
than the size of the config_autoBrightnessLevels array:
* config_autoBrightnessKeyboardBacklightValues
* config_autoBrightnessLcdBacklightValues
* config_autoBrightnessButtonBacklightValues

Otherwise the PowerManageService will get exception
when getting the max value of them.
This is aligned with i.MX53 SMD config.

Change-Id: I5c16f6d435450a67ac220a0efd651e548355abd6
Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 .../overlay/frameworks/base/core/res/res/values/config.xml    | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml b/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
index ef632cb..b9ea4cd 100755
--- a/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
@@ -119,6 +119,7 @@
         <item>240</item>
         <item>250</item>
         <item>255</item>
+        <item>255</item>
     </integer-array>
 
     <!-- Array of output values for button backlight corresponding to the LUX values
@@ -126,10 +127,11 @@
          than the size of the config_autoBrightnessLevels array.
          This must be overridden in platform specific overlays -->
     <integer-array name="config_autoBrightnessButtonBacklightValues">
-        <item>255</item>
-        <item>255</item>
-        <item>255</item>
-        <item>255</item>
+        <item>0</item>
+        <item>0</item>
+        <item>0</item>
+        <item>0</item>
+        <item>0</item>
         <item>0</item>
         <item>0</item>
         <item>0</item>
@@ -149,6 +151,7 @@
         <item>0</item>
         <item>0</item>
         <item>0</item>
+        <item>0</item>
     </integer-array>
 
     <!-- Base "touch slop" value used by ViewConfiguration as a
-- 
1.8.0

